docs/dev/json.rst

```diff
@@ -177,7 +177,10 @@ element                 description
 ``isrc``                The ISRC code associated with the recording.
 ``spotify_id``          The Spotify track URL associated with this recording.  e.g.: http://open.spotify.com/track/1rrgWMXGCGHru5bIRxGFV0
 ``tags``                A list of user defined tags to be associated with this recording. These tags are similar to last.fm tags. For example, you have apply tags such as ``punk``, ``see-live``, ``smelly``. You may submit up to :data:`~webserver.views.api.MAX_TAGS_PER_LISTEN` tags and each tag may be up to :data:`~webserver.views.api.MAX_TAG_SIZE` characters large.
-``listening_from``       The source of the listen, i.e the name of the client or service which submits the listen.
+``listening_from``      The name of the client which submits the listen.
```

Comment:
Does a website count as a client? Not something that would be mentioned in these docs but what about the importers we run?

Reply:
to me, a client is a piece of software that knows how to send a POST to api.listenbrainz.org/1/submit-listen. If that's a webapp that has a musicplayer _and_ submitter built in, then sure. In the case of the spotify writer I think it's close enough that we can use "spotify". Do you have a specific example of a "website" that doesn't fit these criteria?

Reply:
I see makes sense. I don't have an example but consider the case the case of a website that hosts its own music, should both source and listening_from be sent or source be present?

Also, What do you think about adding an examples section after parameters, showing how to decide what values to use in which fields?